Amazon.QLDB.Driver.IntegrationTests/StatementExecutionTests.cs

```diff
@@ -830,17 +828,25 @@ public void Execute_ExecutionMetrics()
             qldbDriver.Execute(txn =>
             {
                 var result = txn.Execute(selectQuery);
+                Driver.IOUsage? ioUsage = null;
+                Driver.TimingInformation? timingInfo = null;
+                long readIOs = 0;
+                long processingTime = 0;
 
                 foreach (IIonValue row in result)
                 {
-                    var ioUsage = result.GetConsumedIOs();
-                    var timingInfo = result.GetTimingInformation();
-
-                    Assert.IsNotNull(ioUsage);
-                    Assert.IsNotNull(timingInfo);
-                    Assert.IsTrue(ioUsage?.ReadIOs > 0);
-                    Assert.IsTrue(timingInfo?.ProcessingTimeMilliseconds > 0);
+                    ioUsage = result.GetConsumedIOs();
+                    if (ioUsage != null)
+                        readIOs += ioUsage.Value.ReadIOs;
+                    timingInfo = result.GetTimingInformation();
+                    if (timingInfo != null)
+                        processingTime += timingInfo.Value.ProcessingTimeMilliseconds;
                 }
+
+                Assert.IsNotNull(ioUsage);
+                Assert.IsNotNull(timingInfo);
+                Assert.IsTrue(readIOs > 0);
```

Comment:
Please assert this is 1092 and add a comment saying this value comes from the statement that performs self joins on a table.

Reply:
This needs to be

```
Assert.AreEqual(1092, readIOs);
```

Reply:
Addressed